src/assets/scripts/client/speech.js

```diff
@@ -21,12 +21,33 @@ export const speech_init = () => {
     }
 };
 
+export const randomize_pilot_voice = () => {
+    const voices = [
+        'Alex',
+        'Daniel',
+        'Fiona',
+        'Google US English',
+        'Google UK English Female',
+        'Google UK English Male'
+    ];
+
+    const voice = voices[Math.floor(Math.random() * voices.length)];
+    const pitch = (Math.random() * (0.9 - 1.125) + 1.125).toFixed(1);
+    const rate = (Math.random() * (1 - 1.125) + 1.125).toFixed(3);
```

Comment:
Please break out these magic numbers into well-labeled variables so we can understand the math by reading the variable names and how they're used.